### PR TITLE
Improve account sidebar and content

### PR DESCRIPTION
**Pull request by lauraluiz**
While implementing the checkout page, I found out the way to implement children routes, such as any section inside the user account. I apply it here and took some time to improve the style too. I think it looks much better now :bowtie:

### CODE REVIEW

**Comment by plaskevich** [comment]
Looks really nice!
But why does the `.my-orders-order-dark-box` have `padding: 15px 0;`?
I think it would look much better if it was `padding: 15px;`

File: src/components/useraccount/userdetail/TabPersonalDetails.vue [hunk 1]
```diff
@@ -76,7 +75,7 @@ export default {
 <i18n>
 en:
   welcomeBack: "Welcome back, {name}"
```
Comment (zeina-idris) [inline src/components/useraccount/userdetail/TabPersonalDetails.vue]:
```suggestion
  welcomeBack: "Hello, {name}"
```

File: src/components/useraccount/userdetail/TabPersonalDetails.vue [hunk 2]
```diff
@@ -88,7 +87,7 @@ en:
     link: "Shop All"
 de:
   welcomeBack: "Willkommen zurück, {name}"
```
Comment (zeina-idris) [inline src/components/useraccount/userdetail/TabPersonalDetails.vue]:
```suggestion
  welcomeBack: "Hallo, {name}"
```

**Comment by lauraluiz** [comment]
Good catch @plaskevich ! The reason is because my screen ignores the grey color and I didn't see the boxes 😅

File: src/components/useraccount/userdetail/TabPersonalDetails.vue (hunk 1 above)
Comment (zeina-idris) [inline src/components/useraccount/userdetail/TabPersonalDetails.vue]:
```suggestion
  welcomeBack: "Hello, {name}"
```

File: src/components/useraccount/userdetail/TabPersonalDetails.vue (hunk 2 above)
Comment (zeina-idris) [inline src/components/useraccount/userdetail/TabPersonalDetails.vue]:
```suggestion
  welcomeBack: "Hallo, {name}"
```

**Pull request merged**